### ISSUE TITLE
How to use it on ionic existing project

### ISSUE THREAD

**Issue by jpujol880807** [0nza1101/ionic3-leaflet-mbtiles#4]
I have this configuration 
ionic/app-scripts : 3.1.2
Cordova Platforms  : android 6.3.0
Ionic Framework    : ionic-angular 3.9.2
Node: 6.11.5
Npm: 3.10.10
System: Windows 10
I'm trying to use the npm package on existing project and it works well on the browser, but it doesn't render the map on android devices, i'm missing some step to make it work?
I added 
import 'promise-polyfill';
import 'whatwg-fetch';
and installed them with npm

**Comment by 0nza1101** [comment]
Hello,
I tested the sample with your configuration seems working without --prod to ionic cordova build android. I will investigate on this issue.

EDIT: I found the problem, the npm packages that I made were using sql.js 0.4.0 I updated it to 0.5.0 but another bug came from this version webpack was throwing a heap out of memory error when building with --prod so I found a fix, this issue is related to webpack/ionic-app-script : [https://github.com/ionic-team/ionic-app-scripts/issues/1036](here).

I prefer adding `"config": {
    "ionic_source_map_type": "cheap-source-map"
  }` to the package.json than using `node --max-old-space-size=8192` in every build.

In definitive update the npm package `npm i leaflet-tilelayer-mbtiles-ts@latest` and add : 
`"config": {
    "ionic_source_map_type": "cheap-source-map"
  },` to your package.json
